marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/test-data/mappings/OrdersMapping/OrdersMapping-4.mapping.json

```diff
@@ -0,0 +1,37 @@
+{
+  "lang": "zxx",
+  "name": "OrdersMapping",
+  "description": "The structured properties are mapped to invalid properties",
+  "version": 4,
+  "targetEntityType": "http://marklogic.com/data-hub/example/OrderType-0.0.1/OrderType",
+  "sourceContext": "/",
+  "properties": {
+    "id": {
+      "sourcedFrom": "id"
+    },
+    "purchaseDate": {
```

Comment:
Do we need anything more than id and customer here? If not, let's remove these other mapping expressions, as they complicate the test. We want the bare minimum of everything to isolate the problem and verify it's fixed.

Reply:
"items" is of type 'array'. I want to make sure it works fine as well.